nvme-print.h

```diff
@@ -83,6 +83,8 @@ struct print_ops {
 	void (*lba_range)(struct nvme_lba_range_type *lbrt, int nr_ranges);
 	void (*lba_status_info)(__u32 result);
 	void (*d)(unsigned char *buf, int len, int width, int group);
+	void (*show_init)(void);
```

Comment:
Let this function return an opaque pointer

Reply:
I see.